packages/opentelemetry-tracing/src/export/BatchSpanProcessor.ts

```diff
@@ -56,7 +55,6 @@ export class BatchSpanProcessor implements SpanProcessor {
   onStart(span: Span): void {}
 
   onEnd(span: Span): void {
-    if (span.context().traceFlags !== TraceFlags.SAMPLED) return;
```

Comment:
+1

Reply:
I added this superfluous check in https://github.com/open-telemetry/opentelemetry-js/pull/218, thanks for fixing it :)